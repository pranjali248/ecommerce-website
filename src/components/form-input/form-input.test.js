import React from "react";
import { shallow } from "enzyme";
import FormInput from "./form-input.component";

describe("Testing form-input component", () => {
  it("Shallow snapshot testing of form-input component", () => {
    expect(shallow(<FormInput />)).toMatchSnapshot();
  });
});
