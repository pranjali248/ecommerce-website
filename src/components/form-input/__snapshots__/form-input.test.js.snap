// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing form-input component Shallow snapshot testing of form-input component 1`] = `ShallowWrapper {}`;
